tests/test_segment.py

```diff
@@ -99,11 +99,20 @@ def test_folder(tmpdir, num_workers, multiprocessing):
         os.path.join(path, f'file{n}.wav') for n in range(3)]
     for file in files:
         af.write(file, SIGNAL, SAMPLING_RATE)
-    result = model.process_folder(path)
+    result = segment.process_folder(path)
     assert all(result.levels[0] == files)
     assert all(result.levels[1] == INDEX.levels[0])
     assert all(result.levels[2] == INDEX.levels[1])
 
+    # non-existing folder
+    with pytest.raises(FileNotFoundError):
+        segment.process_folder('bad-folder')
+
+    # empty folder
+    root = str(tmpdir.mkdir('empty'))
+    y = segment.process_folder(root)
+    pd.testing.assert_index_equal(y, audformat.filewise_index())
```

Comment:
And here we have an index:
```suggestion
    index = segment.process_folder(root)
    pd.testing.assert_index_equal(index, audformat.filewise_index())
```